.github/workflows/codecov.yml

```
@@ -14,13 +14,18 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         php: ['8.2', '8.3', 8.4, '8.4']
```

actionlint (1.7.11)

[error] duplicate value "8.4" is found in matrix "php". the same value is at line:16,col:29

(matrix)